src/uuid/rng/init.lua

```diff
@@ -78,7 +78,42 @@ function rng.luasystem()
   return sys.random
 end
 
+----------------------------------------------------------------------------
+-- Returns an rng that implements Windows' RtlGenRandom function which is
+-- a good source of randomness on all modern versions of Windows.
+-- Requires LuaJIT or a LuaJIT-compatible FFI library.
+-- @treturn function A function that returns `n` random bytes, signature: `byte_string, err = func(n)`
+-- @usage
+-- local uuid = require "uuid"
+-- uuid.set_rng(uuid.rng.win_ffi())
+function rng.win_ffi()
+  local ok, ffi = pcall(require, "ffi")
```

Comment:
I think the first check should be if we're running Windows. Not loading any ffi defs if we're not going to use them anyway